games/u.yaml

```
@@ -239,6 +239,26 @@
   updated: '2019-12-13'
   images:
   - 'https://github.com/krzys-h/UndertaleModTool/raw/master/screenshot.png'
+  
+- name: UnderworldExporter
+  type: remake
+  originals:
+  - Ultima Underworld
+  - 'Ultima Underworld II: Labyrinth of Worlds'
+  repo: 'https://github.com/hankmorgan/UnderworldExporter/'
+  development: active
```

```suggestion
  development: sporadic
```
last commit was 2 months ago